root/dataframe/CMakeLists.txt

```diff
@@ -235,10 +235,18 @@ ROOTTEST_ADD_TEST(read_leaves_nodict
                   EXEC ./read_leaves_nodict
                   DEPENDS ${GENERATE_EXECUTABLE_TEST} read_leaves)
 
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
This is technically a good test ... for now.  Namely this will fails the intent for a standalone `roottest` used to test an installed version of ROOT.   This will not longer be a problem when we merge the 2 repository .... in a few months.